content.tex

```diff
@@ -24,17 +24,20 @@ \section{Types of Out Of Hours Support}
 
 Remote support will be more common than in-person support.  Remote support can be provided by DevOps, software or hardware personnel.  This document refers to this type of work as remote `On-Call' out of hours support. Compensation is provided for On-Call support for the duration of the On-Call hours assigned and additionally for the hours spent on fault resolution.  Please see the Allowance sections below for more details.
```

Comment:
You mention and allowance section here and in the next section, except there doesn't appear to be an allowance section?